cli/src/config.rs

```diff
@@ -0,0 +1,131 @@
+use color_eyre::Result;
+use serde::{de::DeserializeOwned, Deserialize};
+use serde_with::{serde_as, DisplayFromStr};
+use std::{fs::File, io::BufReader, net::SocketAddr, path::PathBuf};
+use tower_http::cors::CorsLayer;
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct Timeout(u64);
+#[derive(Debug, Clone, Deserialize)]
+pub struct CorsAllowAll(bool);
+#[derive(Debug, Clone, Deserialize)]
+pub struct LogAmount(usize);
+#[derive(Debug, Clone, Deserialize)]
+pub struct LogSize(u64);
```

Comment:
These wrappers are not needed. They just clutter the code

Reply:
The way `serde` works is that you need to provide some kind of default function if you want to assign a default value during the deserialization. If none were provided, 'Default::default()' would be called, and it seemed convenient to use wrappers to define Default Trait on them. I'll wait until additional confirmation to remove those structures, if my logic appears to be faulty.

Reply:
The convenient way to provide default value for field is to use the attribute `#[serde(default = ...)]` (https://serde.rs/field-attrs.html#default--path). Example in Bob: https://github.com/qoollo/bob/blob/master/bob-common/src/configs/node.rs#L248